components/ui/AppPageHeaderWithImage.vue

```diff
@@ -2,7 +2,7 @@
   <header class="app-page-header-with-img">
     <div class="bx--grid">
       <div v-if="backLink" class="bx--row app-page-header-with-img__back-link">
-        <AppNavBackLink v-bind="backLink" />
+        <UiAppNavBackLink v-bind="backLink" />
```

Comment:
`v-bind` here

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2980/commits/eef6c40a534c38b7a1e73640346d6f0071d25be8.